src/Mod/Sketcher/Gui/CommandSketcherBSpline.cpp

```diff
@@ -23,6 +23,7 @@
 
 #include "PreCompiled.h"
 #ifndef _PreComp_
+# include <Inventor/nodes/SoPickStyle.h>
```

Comment:
Do you need this header?

Reply:
I used whatever made sense from CommandCreateGeo. Please suggest a more appropriate header if  it exists.

Reply:
Looked up a little and ended up using `Inventor/SbString.h`.